src/components/sections/secondary-banner.json

```diff
@@ -0,0 +1,72 @@
+{
+  "collectionName": "components_sections_secondary_banners",
+  "info": {
+    "displayName": "secondaryBanner",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "name": {
+      "type": "string",
+      "unique": false,
+      "maxLength": 256,
+      "required": true
+    },
+    "title": {
+      "type": "string",
+      "unique": false,
+      "maxLength": 256
+    },
+    "titleHTMLTag": {
+      "type": "enumeration",
+      "enum": [
+        "H1",
+        "H2",
+        "H3",
+        "H4",
+        "H5"
+      ],
+      "default": "H2"
+    },
+    "text": {
+      "type": "text"
+    },
+    "backgroundColor": {
+      "type": "enumeration",
+      "enum": [
+        "white",
+        "black",
+        "background",
+        "primary",
+        "secondary",
+        "tertiary"
+      ],
+      "required": true,
+      "default": "primary"
+    },
+    "image": {
+      "type": "media",
+      "multiple": false,
+      "required": true,
+      "allowedTypes": [
+        "images"
+      ]
+    },
+    "CTALabel1": {
```

Comment:
Suggestion:

Let's reuse Button component here. (primary, secondary)
You can find the example in HeroBanner.

Reply:
The same situation here. What about business requirement? :
'limited to 32 symbols'

There is no such restriction for `Button` - `label` field

Reply:
The same here, let's clarify it with Iryna. Probably, it should be restricted for all buttons.

Reply:
Iryna is fine with the restriction for all buttons
I'll update the code

Reply:
Updated